tasks/blacklight.rake

```diff
@@ -42,11 +42,17 @@ namespace :blacklight do
     if File.exist? EngineCart.destination
       within_test_app do
         system "bundle update"
+
+        system "bundle exec rake blacklight:index:seed" if ENV["DEVELOPMENT_ENVIRONMENT"] == "docker"
+
+        system "bundle exec rails s #{args[:rails_server_args]}"
```

Comment:
This is a duplicate of what is happening on line 61

Reply:
No, @jcoyne the step inside line 61 is happening within a SolrWrapper block.  We are skipping that block when running `docker-compose up`

Reply:
But what about when you aren't in docker-compose?

Reply:
The docker-comopse session sets `DEVELOPMENT_ENVIRONMENT: "docker"` environment variable which changes the behavior of this script so that it skps SolrWrapper (because it's not required) and seeds its per line 46 above.

Reply:
Actually, I see your point... I'll update it.